bundle/DependencyInjection/Configuration.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Netgen\Bundle\IbexaSearchExtraBundle\DependencyInjection;
 
+use Ibexa\Contracts\Core\Repository\LanguageService;
```

Comment:
Not used.

Reply:
resolved in aa257c2d88b8e45a1dfeaaf236b3e57cede8c6bf